todo_app/tests_e2e/test_mongo.py

```diff
@@ -16,10 +19,14 @@
 @pytest.fixture(scope="module")
 def app_with_temp_board():
     load_dotenv(override=True)
-    board_id = create_trello_board()
-    default_list_id = get_default_list(board_id)
-    os.environ["TRELLO_BOARD_ID"] = board_id
-    os.environ["TRELLO_DEFAULT_LIST_ID"] = default_list_id
+    mongo_string = os.environ["MONGO_CONNECTION_STRING"]
+    client = pymongo.MongoClient(mongo_string)
+    db = client["to_do_test"]
+    lists = ["to-do", "doing", "done"]
+    for lst in lists:
```

Comment:
It's worth noting that this isn't _important_, Mongo will happily just create the new collections at run time, but equally nothing wrong with it!

Reply:
While Mongo will do that, some of my view logic (specifically the dropdown for selecting where to move a ticket) does rely on these three lists existing, so if I don't make them/ensure they exist, the view doesn't see them